cmd/webpkgserver/webpkgserver.example.toml

```diff
@@ -202,6 +200,94 @@
+# to handle certificate renewal. Automatic certificate renewal is enabled
+# in webpkgserver via `SXG.ACME.Enable`. Setting this field to true
+# will enable webpkgserver to automatically request certificate renewals
+# whenever it has determined that the current certificate is expired or about to
+# expire.
+#
+# If you set up more than one instance of webpkgserver, the recommended practice
+# is to have only one instance handle automatic certificate renewals and let
+# other instances reload the fresh certificate from the disk. It allows
+# certificates to be shared on all webpkgserver instances (which may help
+# minimize the monetary cost, subject to Terms of Service set by your
+# Certificate Authority) and prevents webpkgserver from bombarding your
+# Certificate Authority with certificate requests.
+#
+# For the full ACME spec, see:
+#   https://tools.ietf.org/html/rfc8555 
+[SXG.ACME]
+  # Enable webpkgserver to attempt to auto renew certificates using
+  # ACME.
+  #Enable = false 
+
+  # The ACME discovery URL. It is specified by the Certificate Authority that
+  # doles out your certificates. As of August 2020, only DigiCert supports
+  # automatic renewals of signed exchange certificate:
+  # https://docs.digicert.com/certificate-tools/acme-user-guide/acme-directory-urls-signed-http-exchange-certificates/
+  # Required when ACME is enabled.
+  DiscoveryURL = '<Your Discovery URL>'
+
+  # The email address registered to the Certificate Authority for your signed
+  # exchange certificates.
+  # Required when ACME is enabled.
+  Email = 'user@example.com'
+
+  # For the remaining configuration items, it is important to understand the
+  # different challenges employed as part of the ACME protocol:
+  # https://ietf-wg-acme.github.io/acme/draft-ietf-acme-acme.html#identifier-validation-challenges
+  # https://letsencrypt.org/docs/challenge-types/
+  # https://certbot.eff.org/docs/challenges.html
+  #
+  # Note that you do not need to configure all of these challenges. It is
+  # typically sufficient to configure just one of them. If more than one method
+  # is configured, the go-acme/lego library will decide pick one of them for
+  # primary use and use other settings as backup. For wildcard certificates,
+  # nevertheless, DNSProvider is the only supported method of validation, and
+  # others cannot be used. See DNSProvider for more detail.
+
+  # The http server root directory where the ACME http challenge token should
+  # be deposited. Note that you may need to do some configuration work to
+  # get this setup to work where multiple instances of webpkgserver is running.
+  # For example:
+  # https://community.letsencrypt.org/t/how-to-nginx-configuration-to-enable-acme-challenge-support-on-all-http-virtual-hosts/5622/3
+  HTTPWebRootDir = '/path/to/www_root_dir'
+
+  # The port used by the webpkgserver to respond to the HTTP challenge
+  # issued as part of ACME protocol. Note that if your setup only opens up
```

Comment:
Thanks for the update, but I found I still hadn't understood "if your setup only opens up certain ports."

Reply:
Clarified.